src/moai-android-adcolony/host.cpp

```diff
@@ -1,6 +1,16 @@
 // Copyright (c) 2010-2011 Zipline Games, Inc. All Rights Reserved.
 // http://getmoai.com
 
+// From moai-box2d's pch.h
+#include <moai_config.h>
+#include <zl-util/headers.h>
+#include <moai-core/headers.h>
+#include <moai-sim/headers.h>
```

Comment:
Replace all MOAIAdColonyAndroid.cpp with host.cpp.  Had the wrong file that was having the issue.  At least, at this point.  It may be that the gradle build under Android Studio may need similar conditional compilation in MOAIAdColonyAndroid.cpp.